brainglobe_utils/general/numerical.py

```diff
@@ -3,9 +3,17 @@
 
 def is_even(num):
     """
-    Returns True if a number is even
-    :param num:
-    :return:
+    Returns True if a number is even.
```

Comment:
```suggestion
    Returns True if the non-zero input number is even.
```

I have _no idea_ why we throw an error if the input is zero (since 0 is a perfectly fine even number) but I presume that this function will be used further up the dependency chain in which this case might be important?

Either way, it probably needs a `Raises` section added too if we're being docstring-precise.

Reply:
That is weird - not sure why this is needed! I added your suggestion + a `Raises` section to indicate zero isn't accepted